src/Plugins/ComplexCore/src/ComplexCore/Filters/ScalarSegmentFeatures.cpp

```diff
@@ -2,26 +2,28 @@
 
 #include <chrono>
 
-#include "complex/Common/StringLiteral.hpp"
-#include "complex/DataStructure/DataArray.hpp"
 #include "complex/DataStructure/DataStore.hpp"
 #include "complex/DataStructure/Geometry/AbstractGeometryGrid.hpp"
 #include "complex/Filter/Actions/CreateArrayAction.hpp"
 #include "complex/Parameters/ArrayCreationParameter.hpp"
 #include "complex/Parameters/ArraySelectionParameter.hpp"
 #include "complex/Parameters/BoolParameter.hpp"
-#include "complex/Parameters/DataGroupSelectionParameter.hpp"
-#include "complex/Parameters/DataPathSelectionParameter.hpp"
 #include "complex/Parameters/NumberParameter.hpp"
 
 using namespace complex;
 
+#define CX_DEFAULT_CONSTRUCTORS(className)                                                                                                                                                             \
+  className(const className&) = delete;                                                                                                                                                                \
+  className(className&&) noexcept = delete;                                                                                                                                                            \
+  className& operator=(const className&) = delete;                                                                                                                                                     \
+  className& operator=(className&&) noexcept = delete;
+
 namespace
 {
 using FeatureIdsArrayType = Int32Array;
 using GoodVoxelsArrayType = UInt8Array;
 
-static inline constexpr StringLiteral k_CompareFunctKey = "Compare Function";
+inline constexpr StringLiteral k_CompareFunctKey = "Compare Function";
```

Comment:
```suggestion
constexpr StringLiteral k_CompareFunctKey = "Compare Function";
```
`inline` doesn't do anything when directly used in a `cpp` file.